src/components/LuxInputButton.vue

```diff
@@ -207,7 +206,6 @@ export default {
   &.solid {
     background: $color-bleu-de-france;
     color: $color-white;
-    &:hover,
```

Comment:
On hover in light theme we would want
&:hover {
text-decoration-color: var(--color-princeton-orange-on-white);
}